### PR TITLE
Added relying parties that need signed assertions and fixed one deprecated property.

### PR DESCRIPTION
**Pull request by mar235av**

### CODE REVIEW

File: conf/relying-party.xml [hunk 1]
```diff
@@ -97,7 +97,7 @@
         </bean>
 
         <!-- needing signed assertions -->
-        <bean parent="RelyingPartyByName" c:relyingPartyIds="#{ {'https://az1.qualtrics.com','https://saml.enplug.com','https://uweducation.communityforce.com','https://uweducationstaging.communityforce.com','https://login.apogeenet.net/sp','https://pvs.education.uw.edu','https://app.yodeck.com/api/v1/account/metadata/','https://sso.pitchbook.com/sso/saml/sp/metadata','https://uw.creatcom.washington.edu/DSF/'} }">
+        <bean parent="RelyingPartyByName" c:relyingPartyIds="#{ {'https://az1.qualtrics.com','https://saml.enplug.com','https://uweducation.communityforce.com','https://uweducationstaging.communityforce.com','https://login.apogeenet.net/sp','https://pvs.education.uw.edu','https://app.yodeck.com/api/v1/account/metadata/','https://sso.pitchbook.com/sso/saml/sp/metadata','https://uw.creatcom.washington.edu/DSF/',''https://uwm.creatcom.washington.edu/DSF/'} }">
```
Comment (jprosser) [inline conf/relying-party.xml:100]: i see an extra ' in the line there

Comment (mar235av) [reply conf/relying-party.xml]: Oops. Fixed.

**Review by jprosser — APPROVED** [review]
Looks good, thanks!

**Comment by mar235av** [comment]
I added three commits to this pull request:
Two commits added two more RPs that also need signed assertions.
While doing that, I noticed a warning for a deprecated property in the same file, so I fixed that by replacing the property name with its designated replacement per Shibboleth documentation.

**Review by jprosser — APPROVED** [review]
Looks good to my eye at least!

**Pull request merged**